endpoint.go

```diff
@@ -555,33 +445,6 @@ func (ep *endpoint) Delete() error {
 	return nil
 }
 
-func (ep *endpoint) Statistics() (map[string]*sandbox.InterfaceStatistics, error) {
```

Comment:
As you found out, the current Statistics() under Endpoint was incorrectly returning the stats for all the interfaces of the attached sandbox, even the interfaces which may connect the sandbox to other endpoints.

So I feel it is better that the current implementation and API is moved under Sandbox (at least it will return the expected behavior, also docker daemon looks for that, stats for all interfaces in a container), and later I can add a proper Statistics() API under Endpoint which would return the stats only for the interfaces which belong to the Endpoint.